src/backend/Csrs.Api/Features/Accounts/NewAccountAndFile.cs

```diff
@@ -91,7 +100,16 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                     dynamicsParty.SsgBceidLastUpdate = DateTimeOffset.Now;
                     dynamicsParty.Statuscode = 1;
 
-                    dynamicsParty = await _dynamicsClient.Ssgcsrsparties.CreateAsync(body: dynamicsParty, cancellationToken: cancellationToken);
+                    try
+                    {
+                        dynamicsParty = await _dynamicsClient.Ssgcsrsparties.CreateAsync(body: dynamicsParty, cancellationToken: cancellationToken);
+                    }
+                    catch (Exception ex)
+                    {
+                        _logger.LogError(ex, "An exception occurred while inserting initial party, file creation will be aborted");
+                        throw;
```

Comment:
I understand that it is happening during initial phase and partyid is unavailable. Is there any other relevant ID (not sensitive) that could be added in the log message?
Thanks!